### ISSUE TITLE
No Validation Set

### ISSUE THREAD

**Issue by 07Agarg** [kekmodel/FixMatch-pytorch#43]
Hi, For supervised and semi-supervised methods, it is generally advised to use a separate validation set. From the code, it looks like the best test set accuracy is reported. 

Is there any specific reason that a separate validation set is not required.

**Comment by kekmodel** [comment]
I focused on reproducing the performance of the paper.
